src/menu/ui_components/tabs.c

```diff
@@ -0,0 +1,15 @@
+#include "../ui_components.h"
+
+
+
+static const char* tabs[3] =
+{
+    "Browser",
```

Comment:
```suggestion
    "Files",
```
Or something like "File Explorer"

Reply:
Perhaps also consider which should be the default Tab... since favorite might be perefereable  (also consider as a config option).

Reply:
a config option for what tab it brings up first could be an idea.
but I'd leave it with the file browser as the default, it feels what people would expect.